### DISCUSSION TITLE
Add option show hidden folders

### DISCUSSION THREAD

**Discussion by DebugBill** [LesFerch/WinSetView#110] [General]
Thanks for this tool.  I really wonder why this is not a standard feature in Windows, I've been waiting for it for years..

I'm happy to confirm it works not only on Window XX, but also on servers which is my main use. 

One additional request though: would it be possible to add in the options the "show hidden folders" checkbox? I always check this one too...

Thanks again.

**Comment by LesFerch** [comment]
Added as of version 2.97 and above.
